committee-steering/governance/annual-reports.md

```diff
@@ -21,83 +21,87 @@ should be highlighted
 ## Reporting Process  
 
 Chairs and Organizers are responsible for compiling a yearly public report but
-may be completed with the help of members of that group. Chairs and Organizers
+may be completed with the help of members of that group. Groups are encouraged
+to make this an agenda item of topic to discuss together. Chairs and Organizers
 ensure that reports are complete, accurate, and submitted to the Steering
 Committee.
 
+### 2021 and on:
+1. Steering Committee liaison reaches out to group leads to kick off
+2. Chairs/Organizers will:
+- share with their groups (see tips below) and begin to 
+compile the doc due at the end of February 
+- use this [question template]
+- submit a pull request to the groups documentation in the community repo
+- if there are private matters to address with Steering before submitting the 
+pull request, share a copy with your laison and wait until comments are
+addressed before submitting.
+3. The March Chairs, Tech Leads, and Organizers meetings will be used as follow 
+up for the community groups that have questions from/to Steering and a 
+retrospective
+4. The Steering Committee liaison will work directly with groups that have 
+follow up items and update Steering during regular monthly meetings. The liaison
+will also coordinate time to follow up with the chairs on their individual roles
+We'd like to hear about the following:  
+- goals you have for the group, your experience in the role so far - challenges,
+ wins, things you didn’t know before but wish you did, 
+- continue in the role or help finding a replacement, 
```

Comment:
> help finding a replacement

seems like the best group to do this would be the sig/wg itself, rather than steering